servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -95,7 +96,11 @@
   private void registerWithMaster() throws IOException {
     BlockStoreMeta storeMeta = mBlockDataManager.getStoreMeta();
     try {
-      mMasterClient.register(WorkerIdRegistry.getWorkerId(), storeMeta.getCapacityBytesOnTiers(),
+      StorageTierAssoc storageTierAssoc =
+          new StorageTierAssoc(WorkerContext.getConf(), Constants.WORKER_TIERED_STORAGE_LEVELS,
+              Constants.WORKER_TIERED_STORE_LEVEL_ALIAS_FORMAT);
```

Comment:
But wouldn't it change if we changed any of the tieredstore properties in the WorkerContext configuration?